CWE-89/doctrine-dbal-dangerous-query.php

```
@@ -0,0 +1,57 @@
+    {
+        $conn = $this->getEntityManager()->getConnection();
+
+// {fact rule=sql-injection@v1.0 defects=1}
+        // ruleid: doctrine-dbal-dangerous-query
+        $query = $conn->createQuery("SELECT u FROM User u WHERE u.username = '" . $_GET['username'] . "'");
+// {/fact}
+        $data = $query->getResult();
+        return $data;
+    }
+
+    public function okTest1(int $price): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$price'. (undefined)

(UnusedFormalParameter)